packages/start/src/client/serialization.tsx

```diff
@@ -111,20 +111,17 @@ export function afterHydrate({ router }: { router: AnyRouter }) {
       }
     }
 
-    const meta =
-      match.status === 'success'
```

Comment:
is the status not relevant anymore?

Reply:
No anymore, since that was only dependent for the loaderData.
Now the loaderData is always possibly undefined.